src/main/scala/com/gd/twitteranalytics/TransformTweets.scala

```diff
@@ -0,0 +1,31 @@
+package com.gd.twitteranalytics
+
+import java.util.{Date => JavaDate}
+import java.sql.Timestamp
+
+import com.gd.twitteranalytics.util.ReadTwitterConf
+import org.apache.spark.streaming.dstream.DStream
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.apache.spark.sql.functions.date_format
+import twitter4j.Status
+import org.apache.spark.rdd.RDD
+
+object TransformTweets{
```

Comment:
object/ class names should be nouns , not verbs. 
e.g. 
* TransformTweets -> TweetsTransformer
* TweetsDataSave --> TweetsDataSaver

Reply:
Fixed !